gym-report-server/src/test/java/com/polot/gym/repository/TrainingReportRepositoryTest.java

```diff
@@ -9,17 +12,32 @@
 
 import java.util.List;
 
-import static org.junit.jupiter.api.Assertions.*;
-
 @DataJpaTest
 class TrainingReportRepositoryTest {
 
     @Autowired
     private TrainingReportRepository trainingReportRepository;
+    private String trainerUsername = "trainer";
+
+    @BeforeEach
+    void setUp() {
+        TrainingReport trainingReport = TrainingReport.builder()
+                .trainerUsername(trainerUsername)
+                .trainerFirstname("firstname")
+                .trainerLastname("lastname")
+                .isActive(true)
+                .year(2023)
+                .month(Month.JANUARY)
+                .date(10)
+                .trainingDuration(2)
+                .type(ReportType.ADD)
+                .build();
+        trainingReportRepository.save(trainingReport);
+    }
 
     @Test
-    void findAllByGroupBy() {
-        List<CustomTrainingReport> reportList = trainingReportRepository.findAllByGroupBy();
-        Assertions.assertThat(reportList).isNotNull();
+    void shouldReturnOneReport() {
```

Comment:
we can check that we've found exactly the same trainingReport which we have in db